### DISCUSSION TITLE
33. CORS에 대해 설명해주세요.

### DISCUSSION THREAD

**Discussion by linirini** [jewelry-box/cs-interview#34] [General]
- CORS에서 서로 다른 출처가 무엇인가요? 서브 도메인은 같은 출처인가요?

**Comment by jongmee** [comment]
## CORS란?
Cross Origin Resource Sharing의 약자로 SOP를 지키는 환경에서 다른 출처의 자원을 브라우저가 사용할 수 있게 하는 매커니즘이다. 

## 서로 다른 출처가 무엇인가?
스킴(프로토콜), 도메인(호스트), 포트가 하나라도 다르다면 다른 출처이다. 즉, 서브 도메인끼리는 다른 출처이다.
